.circleci/config.yml

```diff
@@ -14,5 +14,11 @@ jobs:
           name: update common repo
           command: git submodule update --init
       - run: apt update
+      - run: apt install -y zip
       - run: make release
       - run: make
+      - run:
+          name: make driver
+          command: |
+              cd release/sht3x && make && make clean && cd -
+              cd release/shtc1 && make && make clean && cd -
```

Comment:
Why the make clean in between?

Reply:
You mean `make && make clean`? it's to check that the released driver's clean target works